package.json

```diff
@@ -3,16 +3,14 @@
   "version": "1.11.1",
   "description": "A simple project generator",
   "contributors": [
-    "Jamie King  <Jamie.King@aexp.com> (https://github.com/10xLaCroixDrinker)",
-    "Jonathan Adshead <Jonathan.Adshead@aexp.com> (https://github.com/JAdshead)",
-    "Michael Rochester <Michael.Rochester@aexp.com> (https://github.com/code-forger)"
+    "One App Team <oneamex.careers@aexp.com>"
   ],
   "bin": {
     "create-using-template": "bin/index.js"
   },
   "engines": {
-    "node": ">=10.0.0",
-    "npm": ">=6.12.0"
+    "node": ">=18.x.x <=20.x.x",
```

Comment:
Is there anything that's used in this package that prevents us using it in lower node versions ns? For this utility it may not be good to require node 18 to use

Reply:
Without engines-strict this wouldn't actually stop anyone from using it. Just warn on install